app/abilities/person_ability.rb

```diff
@@ -92,6 +93,10 @@ def if_any_writing_permissions
     contains_any?(writing_permissions, user_context.all_permissions)
   end
 
+  def if_any_writing_permissions_or_any_leaded_events
+    if_any_writing_permissions || user_context.events_with_permission(:event_full).present?
```

Comment:
I am unsure if `.any?` would be better here. I tend to prefer ruby builtin methods over rails additions. My preference might be wrong here...